src/app.js

```diff
@@ -156,15 +159,39 @@ var run = function () {
   chromeCloudSync()
 
   // ----------------- editor ----------------------
-
   var editor = new Editor()
-
-  // ----------------- tabbed menu -------------------
+  // -------------- fileExplorer ------------------
+  // @NOTE HTML#ID creates window['<ID name>'] element as a global
```

Comment:
I wonder if we would like to use jQuery, `window.getElementById` or `window.<id>`, because currently it seems we do use a mixture of these.

jQuery (or a more lightweight alternative) seems to be the safest way given we don't need to reinvent the wheel for supporting different browsers.